xdsl/dialects/riscv.py

```diff
@@ -3272,6 +3272,41 @@ def assembly_line(self) -> str | None:
         )
 
 
+# endregion
+
+# region 17 "V" Standard Extension for Vector Operations
+
+# https://riscv.org/wp-content/uploads/2018/05/15.20-15.55-18.05.06.VEXT-bcn-v1.pdf
```

Comment:
As far as I know the vector extension used in Snitch is non-standard (https://iis-git.ee.ethz.ch/smach/smallFloat-spec). Is this PR related to the paper at all, or are you also adding the standard vector extension for other purposes?

Reply:
Ah yes this is purely for the paper, you make a good point, I'll note that.